v22.2/show-backup.md

```diff
@@ -190,6 +190,33 @@ movr          | public             | vehicles                   | table       |
 . . .
 ~~~
 
+### Show a locality-aware backup
+
+To view a [locality-aware backup](take-and-restore-locality-aware-backups.html), pass locality-aware backup URIs to `SHOW BACKUP`:
+
+{{site.data.alerts.callout_info}}
```

Comment:
I suggest moving the note box directly below the heading here (or below the output). The location right now is a bit disruptive to the flow of the section since it sits after the colon that is actually introducing the `SHOW BACKUP` statement block.

Reply:
moved to below heading instead